README.md

```diff
@@ -19,8 +19,16 @@
 <br />
 
 
+### The Emulator
+
 The emulator exposes a gRPC server that implements the Flow Access API, which is designed to have near feature parity with the real network API.
 
+### The Flowser Emulator Explorer
+
+There is also an block explorer GUI for the emulator, that will help you speed up development when using the emulator. 
```

Comment:
```suggestion
There is also a block explorer GUI for the emulator, that will help you speed up development when using the emulator. 
```

Reply:
Maybe add: Flower allows you to visually inspect transactions, contracts, accounts, and their storage.